src/hand.js

```diff
@@ -0,0 +1,32 @@
+
+
+
+export default class Hand {
+  constructor(deck_id) {
+    this.deck_id = deck_id;
+    this.element = document.createElement('row');
+  }
+  draw() {
+    return fetch(`https://card-proxy.herokuapp.com/decks/${this.deck_id}/draw?count=5`)
+      .then((res) => res.json()
+      .then((data) => {
+        this.data = data;
+      }))
+  }
+  selectors = {
```

Comment:
This selectors object should be part of the constructor I think. What's happening right now is that the `selectors` is being created before `this` exists.